app/forms/ConsignmentReferences.scala

```diff
@@ -38,14 +39,41 @@ object ConsignmentReferences {
 
   val allowedReferenceAnswers: Seq[String] = Seq(Ducr, Mucr)
 
-  val mapping = Forms.mapping(
-    "reference" -> requiredRadio("consignmentReferences.reference.empty")
-      .verifying("consignmentReferences.reference.error", isContainedIn(allowedReferenceAnswers)),
-    "referenceValue" -> text()
-      .verifying("consignmentReferences.reference.value.empty", nonEmpty)
-      .verifying("consignmentReferences.reference.value.error", isEmpty or validDucrOrMucr)
-  )(ConsignmentReferences.apply)(ConsignmentReferences.unapply)
+  val mapping = Forms
+    .mapping(
+      "reference" -> requiredRadio("consignmentReferences.reference.empty")
+        .verifying("consignmentReferences.reference.error", isContainedIn(allowedReferenceAnswers)),
+      "referenceValue" -> text()
+        .verifying("consignmentReferences.reference.value.empty", nonEmpty)
+    )(ConsignmentReferences.apply)(ConsignmentReferences.unapply)
 
   def form(): Form[ConsignmentReferences] = Form(mapping)
 
 }
+
+object ConsignmentReferencesForm {
+
+  def bindFromRequest(implicit request: play.api.mvc.Request[_]) = {
```

Comment:
👍

Reply:
Any reason why there isnt an import for `play.api.mvc.Request` rather than having the package prefix?

Reply:
Fixed.  Happy now @ed0906 ?